app/assets/stylesheets/farmbot.css.scss

```diff
@@ -107,20 +113,60 @@ $brown:          #CA8;
   color: $white;
   font-size: 20px;
   text-align: center;
-
+  display: inline-block;
+  font-style: normal;
+  cursor: pointer;
   &:after {
     content: '\f067';
   }
 }
+
+  .farm-designer-body {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .farm-designer-left {
+    order: -1;
+    background-color: $light_green;
+  }
+  .farm-designer-body {
+    flex-direction: row;
+    flex: 1;
+  }
+  .farm-designer-middle {
+    flex: 1;
+    background-color: $brown;
+  }
+  .farm-designer-left, .farm-designer-right {
+    /* 12em is the width of the columns */
+    flex: 0 0 16em;
+  }
+  .farm-designer-right {
+    background-color: $light_purple;
```

Comment:
Name of variable `light_purple` should be written in all lowercase letters with hyphens instead of underscores